docs/tutorials/testing_with_control_center.rst

```diff
@@ -33,6 +33,10 @@ section of our website, and individual links are provided below for convenience:
 
 Hardware Setup
 ==============
+.. warning:: **If you are using a power supply to power your module, it is possible  to damage or destroy your module with aggressive commands.** This is because Vertiq modules can also act as generators. 
+  Power supplies generally cannot absorb the energy generated by the module, while batteries can, which can lead to dangerous voltage spikes when connected to a power supply. 
```

Comment:
In general, power supplies, unlike batteries, cannot absorb the energy generated by the module. As such, aggressive commands can lead ...